python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -91,6 +91,11 @@ def combine_events(
     )
     process_logger.log_start()
 
+    if vp_events.shape[0] == 0 and tu_events.shape[0] == 0:
+        process_logger.add_metadata(total_event_count=0)
+        process_logger.log_complete()
+        return pandas.DataFrame()
```

Comment:
handle the situation where vehicle events and/or trip updates together produce no usable rows from parquet files